### ISSUE TITLE
Development Mode

### ISSUE THREAD

**Issue by Tringlle** [Minehut/Meta#851]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**

**Describe the feature**

There should be a development mode for minehut servers that either keeps it online until toggled off, or just doesnt hibernate it
While its in dev mode, if its online until toggled off, people couldn't join the server until toggled off. If it just doesnt go into hibernation, then thats pretty self explanitory
**How the feature is useful**

Its helpful to people that dont want to loose progress due to server hiberantion.
---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by ahoulton** [comment]
So basically a "file manager" only experience? If so, why? Just use a text editor on your local machine and copy it over when your done.

**Comment by Tringlle** [comment]
> So basically a "file manager" only experience? If so, why? Just use a text editor on your local machine and copy it over when your done.

Not exactly, i imagine it as a testing environment. If its in dev mode, you can access console & edit files only. That way you can test it as you write it.

**Comment by IWolfZZ** [comment]
Theres a command called /whitelist if youre looking for that? I don't get the point of this. Only if your talking about Server maintenance mode?

**Comment by mybeary-afk** [comment]
If you're testing for errors as you're writing it, there are a bunch of resources online that can offer the same result.

https://parser.skunity.com/
https://github.com/SkriptLang/skript-parser

I get that the idea is to completely isolate the testing environment and to allow as much uptime to develop on the server wherever you are, but there is a fact on the matter that this does consume resources; the fact that the server isn't going to hibernate is also going to appeal to a lot of afk farmers as well.

**Comment by Tringlle** [comment]
> If you're testing for errors as you're writing it, there are a bunch of resources online that can offer the same result.
> 
> https://parser.skunity.com/ https://github.com/SkriptLang/skript-parser
> 
> I get that the idea is to completely isolate the testing environment and to allow as much uptime to develop on the server wherever you are, but there is a fact on the matter that this does consume resources; the fact that the server isn't going to hibernate is also going to appeal to a lot of afk farmers as well.

It could use the "Hibernate" plugin which basically lags the server so it uses like 0.1% cpu. This would only be done during dev mode. However yes, its just a testing environment, and yes it is possible in other places.
